src/components/profile/__test__/PersonalDetailsComponent.test.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { shallow } from "enzyme";
```

Comment:
@babul3012 may i as what shallow is for my own knowledge?

Reply:
shallow tests the component without rendering the child components. For more information https://enzymejs.github.io/enzyme/docs/api/shallow.html

Reply:
ah thats cool, cheers for explaining